<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="ByNameRadio">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>By Name</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ByExtensionRadio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>By Extension</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ByDateRadio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>By Date</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="BySizeRadio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>By Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartIndexingButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Indexing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PauseIndexingButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause Ind</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopIndexingButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Ind</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchRequestLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter request</string>
    </property>
   </widget>
   <widget class="QListWidget" name="SearchResultList">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="print">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Event</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>233</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResumeIndexingButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume Ind</string>
    </property>
   </widget>
   <widget class="QLabel" name="indexlabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Indexing file:</string>
    </property>
   </widget>
   <widget class="QLabel" name="searchlabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Searching file:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
